Ball_merge_surface_reconstruction/doc/Ball_merge_surface_reconstruction/Ball_merge_surface_reconstruction.txt

```diff
@@ -7,69 +7,41 @@ namespace CGAL {
-\cgalFigureEnd
-
-\subsection secLocalBMSR Local BallMerge
+The Global variant, by definition, results in a watertight surface (not always manifold) and, hence, is optimal for reconstructing closed surfaces. As the algorithm tries to close the surface, it can withstand missing data up to an extent and can handle little outliers and noise.
 
-\cgalFigureBegin{figBMSRlocalillus,BMSRLocalIllustration.png}
-L.t.r.: Global view with colored Voronoi balls, closeup (blue), reconstruction - global (pink)/local (orange).
-\cgalFigureEnd
+The Local variant, however, is designed for handling open surfaces. As it is lightweight and requires only a single pass over the Delaunay simplices, it is optimal for reconstruction from an extremely huge point cloud.
 
-Although \f$ Global BallMerge\f$ algorithm is well suited for reconstructing watertight surfaces, many real-world scans represent non-watertight models due to extensive missing data and open surfaces - for example,
-in 2D (please note that the core idea of \f$ BallMerge \f$ works in both 2D and 3D), a set of points representing a shape with open curves and junctions is shown in Figure \cgalFigureRef{figBMSRlocalillus}. Though our method could successfully partition the Voronoi balls based on the intersection ratio,
-we do not get the expected result using \f$ Global BallMerge\f$ as it outputs only a single connected component. Therefore, in this local variant of the \f$ BallMerge \f$ algorithm, for any two adjacent \f$d\f$-simplices,
-which are not \f$ \delta\f$-mergeable, the shared \f$(d − 1)\f$-simplex is retained. As expected and can be observed in our experiments, outliers and open surfaces result in very large simplices. Thus, we consider
-elements \f$ \delta\f$-mergeable only if the longest edge of the simplex is smaller than \f$ \frac{1}{\eta}*\beta \f$ (\f$ \eta \f$ is a scaling factor, \f$ \beta \f$ is the length of the bounding box diagonal)
-to remove large simplices (typical for open meshes).
+\subsection secBMSRPar Parameter Settings
 
-In practice, the threshold \f$ \eta = 200\f$ was observed to perform well and can be adjusted - Figure \cgalFigureRef{figBMSRetatuning} shows the effect of varying \f$ \eta \f$.
+The main parameter used in BallMerge is the intersection ratio \f$ > \delta \f$. By definition, the \f$ > \delta \f$ ranges from 0 to 2 - with  \f$ BallMerge(0) \f being empty and  \f$ BallMerge(2) \f containing all the triangles of the Delaunay Triangulation. Experiments show that the \f$ > \delta \f$ value usually lies between 1.5 and 1.95 (with 1.8 or 1.85 giving the best results in most cases). Thanks to the nice property of this intersection ratio, it is quite easy to tune this parameter based on a simple rule: if the reconstructed surface has missing parts, the threshold should be lowered, or if the reconstructed surface has additional parts, the threshold should be increased. 
```

Comment:
I guess there is a dollar missing at the closing  "\f".    Also text inside mathmode should be `\textrm{..}`